src/main/java/com/zendesk/maxwell/schema/columndef/DateTimeColumnDef.java

```diff
@@ -38,10 +35,10 @@ public boolean matchesMysqlType(int type) {
 	private String formatValue(Object value) {
 		if ( value instanceof Long && getType().equals("datetime") )
 			return formatLong((Long) value);
-		else if ( value instanceof Date )
-			return getDateTimeFormatter().format((Date) value);
 		else if ( value instanceof Timestamp )
 			return getDateTimeFormatter().format((Timestamp) value);
+		else if ( value instanceof Date )
```

Comment:
`java.sql.Date` and `java.sql.Timestamp` are both subclasses of `java.util.Date` so the if statement `if ( value instanceof Timestamp )` was previous unreachable.